adage/analyze/models.py

```diff
@@ -218,3 +218,17 @@ def __unicode__(self):
         return "Model: %s, Node: %s, Gene: %s" % (self.node.mlmodel.title,
                                                   self.node.name,
                                                   self.gene.entrezid)
+
+
+class ExpressionValue(models.Model):
+    """
+    This class models the many-to-many relationship between Sample and Gene.
+    Each sample/gene pair has an expression value of floating type.
+    """
+    sample = models.ForeignKey(Sample, on_delete=models.PROTECT)
+    gene = models.ForeignKey(Gene, on_delete=models.PROTECT)
+    value = models.FloatField()
```

Comment:
I don't think gene expression values have units. They are relative measurements. To make it a little more precise, you can say "RMA processed and zero-one normalized gene expression values".